pkg/server/landing.go

```diff
@@ -0,0 +1,47 @@
+package server
```

Comment:
I looked into it, we can't put them in a different folder while keeping them in the server package. We need them in the server package so they can hang off the server (See article). It's annoying but I can't find a way around it.

Reply:
We should have a chat with @mikena-truss  about this

Reply:
+1 on having this in `handlers`. We'll probably want individual handler structs, since they will have their own dependencies/unique fields. You can roughly follow this guide and add/update it as we make changes: https://github.com/trussworks/Engineering-Playbook/blob/master/web/server/go.md.